### ISSUE TITLE
Weight only for people

### ISSUE THREAD

**Issue by hcscysh** [ultralytics/ultralytics#19451]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Is there a model weight specifically for detecting people? I mean not from 80 categories, but only for the category of people, but only for the category of people, which is more robust than the basic weights, such as posture. When testing, it was found that the basic weights did not perform well when detecting lying people.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hcscysh, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com/) for more details, as it contains a wealth of helpful information for using and customizing YOLO models. For model-related inquiries, you may find the [Models and Tasks](https://docs.ultralytics.com/tasks/) section relevant.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

For your custom training ❓ Question, it sounds like you're looking for a fine-tuned model for detecting only "person" objects, potentially trained on a task such as posture or specific orientations like lying down. To achieve this, you may want to explore custom training by using the COCO dataset person class or other posture-specific datasets. Follow our [Custom Training](https://docs.ultralytics.com/guides/model-training-tips/) guide for step-by-step assistance, and please share any additional details, such as your dataset or training logs, to help us provide better guidance.

Join our vibrant community where it suits you best:
- Real-time chat on [Discord](https://discord.com/invite/ultralytics) 🎧
- Detailed discussions on [Discourse](https://community.ultralytics.com/)
- Interesting threads on our [Subreddit](https://reddit.com/r/Ultralytics)

## Upgrade

Ensure you’ve upgraded to the latest version of `ultralytics` to access improvements, features, and bug fixes. You can update in any Python>=3.8 environment like this:

```bash
pip install -U ultralytics
```

## Environments

YOLO models can be used on several platforms with ready-to-use environments:
- **Notebooks** with GPUs: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: [Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: [Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: [Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

When this badge is green, it means all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI tests confirm the correct functioning of YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across platforms like macOS, Windows, and Ubuntu.

This is an automated response, but an Ultralytics engineer will follow up with you soon for additional support. 😊

**Comment by glenn-jocher** [comment]
For **human-only detection** and **pose estimation**, we recommend using our YOLO11-pose models pretrained on COCO with only the 'person' class. These models are optimized for human keypoint detection and may better handle varied postures:

```python
from ultralytics import YOLO
model = YOLO('yolo11n-pose.pt')  # smallest
model = YOLO('yolo11x-pose.pt')  # largest and most accurate
```

For detection-only (no pose), you can use YOLO-World to create a custom people detector:
```python
model = YOLO('yolov8s-world.pt')
model.set_classes(["person"])  # single-class detection
model.save('yolo-person.pt')  # save as custom weights
```

If you need improved performance on non-standard postures, we recommend fine-tuning on a custom dataset containing lying poses. The COCO-Pose dataset documentation shows how to format pose data.

All usage requires an AGPL-3.0 license (open source) or Ultralytics Enterprise License for commercial/proprietary use.
